Failure on aarch64-linux [(full log)](https://logs.nix.gsc.io/?key=nixos/nixpkgs.34794&attempt_id=3ce9db6d-1017-4ad4-a0c1-c14e998ad5a0)

<details><summary>Partial log (click to expand)</summary><p>

```
/nix/store/7fxbh1yhagvwbdrmdyyy5ghcjhwjndhs-bash-4.4-p12/bin/bash: tabdialog.ui: command not found
/nix/store/7fxbh1yhagvwbdrmdyyy5ghcjhwjndhs-bash-4.4-p12/bin/bash: arraydialog.ui: command not found
/nix/store/7fxbh1yhagvwbdrmdyyy5ghcjhwjndhs-bash-4.4-p12/bin/bash: letterdialog.ui: command not found
make: *** [Makefile:2528: .ui/ui_letterdialog.h] Error 127
make: *** [Makefile:2522: .ui/ui_arraydialog.h] Error 127
make: *** [Makefile:2525: .ui/ui_tabdialog.h] Error 127
/nix/store/7fxbh1yhagvwbdrmdyyy5ghcjhwjndhs-bash-4.4-p12/bin/bash: quickdocumentdialog.ui: command not found
make: *** [Makefile:2531: .ui/ui_quickdocumentdialog.h] Error 127
builder for '/nix/store/6slmgjqczg5bym7vn4zndmsgxfxpjw0r-texstudio-2.12.6.drv' failed with exit code 2
error: build of '/nix/store/6slmgjqczg5bym7vn4zndmsgxfxpjw0r-texstudio-2.12.6.drv' failed
```
</p></details>

--- a/pkgs/applications/editors/texstudio/default.nix
+++ b/pkgs/applications/editors/texstudio/default.nix
@@ -1,4 +1,4 @@
-{ stdenv, fetchurl, qt4, qmake4Hook, poppler_qt4, zlib, pkgconfig}:
+{ stdenv, fetchurl, qt5, poppler_qt5, zlib, pkgconfig}:
 
 stdenv.mkDerivation rec {
   pname = "texstudio";
@@ -11,8 +11,8 @@ stdenv.mkDerivation rec {
     sha256 = "18rxd7ra5k2f7s4c296b3v3pqhxjmfix9xpy9i1g4jm87ygqrbnd";
   };
 
-  nativeBuildInputs = [ qmake4Hook pkgconfig ];
-  buildInputs = [ qt4 poppler_qt4 zlib ];
+  nativeBuildInputs = [ qt5.qmake pkgconfig ];
+  buildInputs = [ qt5.qtbase qt5.qtscript qt5.qtsvg poppler_qt5 zlib ];
 
   qmakeFlags = [ "NO_APPDATA=True" ];
 
@@ -25,7 +25,7 @@ stdenv.mkDerivation rec {
 	'';
     homepage = http://texstudio.sourceforge.net;
     license = licenses.gpl2Plus;
-    platforms = platforms.linux;
+    platforms = [ "x86_64-linux" ];
     maintainers = with maintainers; [ cfouche ];
   };
 }

